tcl/nsv_dict.tcl

```diff
@@ -82,4 +82,9 @@ namespace eval qc::nsv_dict {
             return [dict get $temp {*}[lrange $keys 1 end]]
         }        
     }
+
+    proc lappend {variable key values} {
+        #| Append the values to the key in an nsv_array
+        nsv_lappend $variable $key {*}$values
```

Comment:
#| Append values to the list corresponding to the dictionary key stored in a nsv_array